src/compiler/transformers/es2015.ts

```diff
@@ -1825,6 +1818,10 @@ namespace ts {
             }
 
             if (isBlock(body)) {
+                // ensureUseStrict is false because no new prologue-directive should be added.
+                // addPrologueDirectives will simply put already-existing directives at the beginning of the target statement-array
+                const statementOffset = addPrologueDirectives(statements, body.statements, /*ensureUseStrict*/ false, visitor);

```

Comment:
I'm trying to create a test and realised I don't know how to get `"use strict"` to be generated anywhere except at the beginning of a file. How can I get it to emit at the beginning of a function?